codegenerator/cli/templates/dynamic/init_templates/shared/package.json.hbs

```diff
@@ -41,15 +41,12 @@
     "ts-mocha": "^10.0.0",
     "ts-node": "10.9.1",
     "typescript": "5.2.2",
+    "chai": "4.3.10",
   {{/if}}    
     "mocha": "10.2.0"
   },
   "dependencies": {
-    {{#if is_typescript}}
-    "chai": "4.3.10",
-    {{/if}}    
-    "envio": "{{envio_version}}",
-    "ethers": "6.8.0"
```

Comment:
I assume it was needed because of `Ethers.ethAddressToString` calls adding "ethers" imports to the handler. Since we moved to Viem and I also prevented the issue with imports, I think it's not needed in the indexer dependencies anymore.

Reply:
We use ethers client for RPC queries at the moment.

Reply:
So we do need it unfortunately. Hoping to refactor this soon.

Reply:
Yes, it's still installed in the generated package